controllers/githubHooks.js

```diff
@@ -136,31 +113,16 @@ var HooksController = {
 
 function handleIssueEvent(body) {
    debug('Webhook action: %s for issue #%s', body.action, body.issue.number);
-   var before = Promise.resolve();
+   // Some of these events are also triggered on pull requests
+   var isPull = body.issue.pull_request;
+
+   var doneHandling = handleLabelEvents(body);
+
    switch(body.action) {
       case "opened":
          // Always do this for opened issues because a full refresh
          // is the easiest way to get *who* assigned the initial labels.
-         return refresh.issue(body.issue.number);
```

Comment:
Should this wait for `doneHandling` to resolve before calling `refreshPullOrIssue`?

It wouldn't change anything right now, since `handleLabelEvents` will return a fulfilled promise for this `body.action`.